system/ee/ExpressionEngine/View/_templates/login.php

```diff
@@ -20,16 +20,20 @@
 		<?=ee()->view->script_tag('jquery/jquery.js')?>
 		<?=ee()->view->script_tag('common.min.js')?>
 		<?=ee()->view->script_tag('vendor/focus-visible.js')?>
-		<script>
-			var eyeOpen = '<?=URL_THEMES_GLOBAL_ASSET.'img/eye-open.svg'?>',
-				eyeClosed = '<?=URL_THEMES_GLOBAL_ASSET.'img/eye-closed.svg'?>'
-		</script>
-		<?=ee()->view->script_tag('cp/login.js')?>
+		
```

Comment:
Just to confirm @intoeetive was the removal of the eye on purpose?

Reply:
Yes - it was moved to external JS file